api/v1/views/places.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+"""
+Cities file
+"""
+from flask import jsonify, request, abort, make_response
+from api.v1.views import app_views
+from models import storage
+from models.state import State
+from models.city import City
+from models.place import Place
+
+
+@app_views.route('/cities/<city_id>/places', methods=['GET'],
+                 strict_slashes=False)
+def get_places(city_id):
+    """ Function that returns a JSON """
+    the_obj = storage.get(City, city_id)
+    if the_obj is None:
+        abort(404)
+    my_list = []
+    for obj in the_obj.places:
+        my_list.append(obj.to_dict())
+    return jsonify(my_list)
+
+
+@app_views.route('/places/<place_id>', methods=['GET'], strict_slashes=False)
+def get_place(place_id):
+    the_obj = storage.get(Place, place_id)
+    if the_obj is None:
+        abort(404)
+    return jsonify(the_obj.to_dict())
+
+
+@app_views.route('/places/<place_id>', methods=['DELETE'],
+                 strict_slashes=False)
+def del_place(place_id):
+    the_obj = storage.get(Place, place_id)
+    if the_obj is None:
+        abort(404)
+    storage.delete(the_obj)
+    storage.save()
+    return jsonify({}), 200
+
+
+@app_views.route('/cities/<city_id>/places', methods=['POST'],
+                 strict_slashes=False)
+def create_place(city_id):
+    try:
+        json_post = request.get_json()
+        the_obj = storage.get(City, city_id)
+        if the_obj is None:
+            abort(404)
+        if not json_post:
+            return make_response(jsonify({'error': 'Not a JSON'}), 400)
```

Comment:
I think we need to use abort 400 and then in the app.py file we make an errorhandler for error 400